install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2798,6 +2804,12 @@ clustermesh:
     # -- Number of replicas run for the clustermesh-apiserver deployment.
     replicas: 1
 
+    # -- lifecycle setting for the apiserver container.
+    lifecycle: {}
+
+    # -- terminationGracePeriodSeconds for the clustermesh-apiserver deployment.
```

No periods for sentence fragments
```suggestion
    # -- lifecycle setting for the apiserver container
    lifecycle: {}

    # -- terminationGracePeriodSeconds for the clustermesh-apiserver deployment
```